pkg/registry/file/watch_test.go

```diff
@@ -238,7 +237,7 @@ func TestFilesystemStoragePublishesToMatchingWatch(t *testing.T) {
 			var ttl uint64 = 0
 			var out runtime.Object
 			for key, object := range tc.inputObjects {
-				s.Create(ctx, key, object, out, ttl)
+				_ = s.Create(ctx, key, object, out, ttl)
```

Comment:
question: is there any reason we assign to a throwaway variable?

Reply:
golangci-lint

Reply:
:ok_hand: